doc/nrf/protocols/matter/getting_started/advanced_kconfigs.rst

```diff
@@ -15,6 +15,23 @@ Defining path to project-specific Matter settings
 You can use the :kconfig:option:`CONFIG_CHIP_PROJECT_CONFIG` Kconfig option to define the path to the configuration file that contains project-specific Matter settings in the form of C preprocessor macros.
 These macros cannot be altered using dedicated Kconfig options.
 
+.. _ug_matter_configuring_optional_ble_advertising:
+
+Bluetooth LE advertising
+========================
+
+The Matter specification requires the accessory device to advertise Matter service over Bluetooth LE for the commissioning purposes.
+By default, the Bluetooth LE advertising start has to be requested by the application (e.g. in result of a button press) and lasts for a maximum duration of 15 minutes.
+This scenario is appropriate for a devices with a high security requirements that should not advertise its service without a direct trigger, for example door locks.
+
+However, you can configure when the device will start and how long it will advertise:
```

Comment:
```suggestion
However, you can configure when the device will start advertising and how long it will advertise with the following Kconfig options:
```